### PR TITLE
Add more time to grace and period

### PR DESCRIPTION
**Pull request by bkiseu**
What does this PR do?
User should be able to set timeouts and grace values for days exceeding 30

Description of Task completed
This period should be extended because it is possible to have cron jobs that run once every 30+ days

What are the relevant pivotal tracker stories?
[Pivotal Tracker](https://www.pivotaltracker.com/story/show/147010559)

### CODE REVIEW

File: static/js/checks.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 $(function () {
 
-    var MINUTE = {name: "minute", nsecs: 60};
+    var MINUTE = {name: "min", nsecs: 60};
```
Comment (bmwachajr) [inline static/js/checks.js:3]: It would be better to maintain the key as minute because min could mean minimum

Comment (bkiseu) [reply static/js/checks.js:3]: cool

Comment (bkiseu) [reply static/js/checks.js:3]: Oh, minute was taking up too much space, that's why i did that

Comment (bkiseu) [reply static/js/checks.js:3]: on the front end that is. When adding the grace period n stuff

File: static/js/checks.js [hunk 2]
```diff
@@ -36,14 +36,17 @@ $(function () {
...
-            '66%': [86400, 86400],
-            '83%': [604800, 604800],
-            'max': 2592000,
+            '20%': [3600, 3600],
+            '45%': [86400, 86400],
+            '63%': [604800, 604800],
+            '80%': [2592000, 2592000],
+            '87%': [3456000, 3456000],
+            '93%': [4320000, 4320000],
+            'max': [5184000, 5184000]
```
Comment (andela-landia) [inline static/js/checks.js:45]: think max should be one number not an array

File: static/js/checks.js [hunk 3]
```diff
@@ -36,14 +36,17 @@ $(function () {
...
         },
         pips: {
             mode: 'values',
-            values: [60, 1800, 3600, 43200, 86400, 604800, 2592000],
+            values: [60, 1800, 3600, 43200, 86400, 604800, 2592000, 5184000],
```
Comment (andela-landia) [inline static/js/checks.js:49]: this should match all values in your range include 3456000, 4320000

**Review by andela-landia — CHANGES_REQUESTED** [review]

File: static/js/checks.js (hunk 2 above)
Comment (bkiseu) [reply static/js/checks.js:45]: fixed

File: static/js/checks.js (hunk 3 above)
Comment (bkiseu) [reply static/js/checks.js:49]: Making them really messes up the slider.

**Review by andela-landia — APPROVED** [review]

**Pull request merged**